docs/pages/versions/v53.0.0/sdk/accelerometer.mdx

```diff
@@ -22,17 +22,18 @@ import { SnackInline } from '~/ui/components/Snippet';
 <SnackInline label="Basic Accelerometer usage" dependencies={['expo-sensors']}>
 
 ```jsx
```

Comment:
To make this example compatible with Snack + TypeScript and run without errors, please update the code language here to `tsx`.